config.py

```diff
@@ -222,6 +222,8 @@ def _checksd_path(directory):
 
 
 def _is_affirmative(s):
+    if s is None:
```

Comment:
`return s is None or isinstance(s, int) and bool(s) or s.lower() in ('yes', 'true', '1')`